### PR TITLE
Release 1.2.0

### PR DESCRIPTION
**Pull request by sysadmind**
Signed-off-by: Joe Adams <github@joeadams.io>

### CODE REVIEW

**Comment by adammike** [comment]
Does this fix the problem with pulling metrics from elasticsearch 7.13? I don't see the notes in the changelog and I'd recommend pulling in that PR before the next release.

**Review by caarlos0 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+## 1.2.0
+
+This release marks the first release under the prometheus-community organization.
+
+* [FEATURE] Added elasticsearch_clustersettings_stats_max_shards_per_node metric
```
Comment (SuperQ) [inline CHANGELOG.md:5]: Usually we include `#XXX` for the PR that implements each feature. GitHub automatically links these, which is handy.

Comment (sysadmind) [reply CHANGELOG.md]: Thanks. This should be fixed now.

**Comment by caarlos0** [comment]
Maybe we could update the repo settings to accept only squash on merge to make it easier to create the changelogs (basically `git log previous_tag...version`)?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+This release marks the first release under the prometheus-community organization.
+
+* [FEATURE] Added elasticsearch_clustersettings_stats_max_shards_per_node metric. #277
+* [FEATURE] Added elasticsearch_indices_shards_store_size_in_bytes metric. #292
+* [FEATURE] Added --es.indices_mappings flag to scrape elasticsearch index mapping stats. #411
+* [FEATURE] Added elasticsearch_indices_mappings_stats_fields metric. #411
+* [FEATURE] Added elasticsearch_indices_mappings_stats_json_parse_failures_total metric. #411
+* [FEATURE] Added elasticsearch_indices_mappings_stats_scrapes_total metric. #411
+* [FEATURE] Added elasticsearch_indices_mappings_stats_up metric. #411
+* [FEATURE] Added elasticsearch_indices_settings_total_fields metric. #411
```
Comment (SuperQ) [inline CHANGELOG.md:12]: This should be one entry.

**Review by SuperQ — APPROVED** [review]
LGTM!

**Pull request merged**